docs/source/status.rst

```diff
@@ -161,6 +161,7 @@ Notes:
 
 * \(1) No support for handshake or DoExchange.
 * \(2) Support using AspNetCore authentication handlers.
+* \(3) Flight integration testing for Rust is still ongoing as of version 3.0.0, see ARROW-8853.
```

Comment:
@carols10cents I haven't ticked anything here, as we haven't merged the flight test PR

Reply:
I don't think we should add information about work-in-progress features here. This page is for end-users.

Reply:
Thanks, I'll remove WIP